promptfoo/e2e/http-provider.yaml

```
@@ -1,12 +1,15 @@
 id: http
 config:
   useHttps: false
-  url: 'http://localhost:8080/api/chat'
+  url: 'http://127.0.0.1:8080/api/chat'
   method: POST
   headers:
-      x-model: '{{model}}' # A way to control offline test execution. Disable on prod
+#      Content-Type: application/json
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)